language/types_values_variables.md

```diff
@@ -142,16 +146,17 @@ may appear more than once in the hierarchy (e.g. a `Scalar` is both `Any` and `D
        |  |     |- (Integer with range inside another Integer)
        |  |  |- Float[from, to]
        |  |     |- (Float with range inside another Float)
+       |  |  |- Timespan
+       |  |  |- Timestamp
```

Comment:
This looks problematic, while they can be involved in arithmetic they are not numeric data types in terms of Data (basically something that can be expressed in Json with only Json data types). There will also be problems if a type accepts Numeric and is not prepared to handle TimeStamp or TimeSpan. Arithmetic rules are also not cumulative. Suggest that they are not considered to be of Numeric type.